ee/clickhouse/models/event.py

```diff
@@ -42,19 +40,18 @@ def create_event(
     if elements and len(elements) > 0:
         elements_chain = elements_to_string(elements=elements)
 
-    pb_event = events_pb2.Event()
-    pb_event.uuid = str(event_uuid)
-    pb_event.event = event
-    pb_event.properties = json.dumps(properties)
-    pb_event.timestamp = timestamp.strftime("%Y-%m-%d %H:%M:%S.%f")
-    pb_event.team_id = team.pk
-    pb_event.distinct_id = str(distinct_id)
-    pb_event.elements_chain = elements_chain
-    pb_event.created_at = timestamp.strftime("%Y-%m-%d %H:%M:%S.%f")
-
-    p = ClickhouseProducer()
-
-    p.produce_proto(sql=INSERT_EVENT_SQL(), topic=KAFKA_EVENTS, data=pb_event)
+    kafka_event = {
+        "uuid": str(event_uuid),
+        "event": event,
+        "properties": json.dumps(properties),
+        "timestamp": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
+        "team_id": team.pk,
+        "distinct_id": str(distinct_id),
+        "elements_chain": elements_chain,
+        "created_at": timestamp.strftime("%Y-%m-%d %H:%M:%S.%f"),
+    }
+
+    ClickhouseProducer().produce(sql=INSERT_EVENT_SQL(), topic=KAFKA_EVENTS_JSON, data=kafka_event)
```

Comment:
Q: Is the protobuf path actually being executed anywhere in prod code?

Reply:
nope, explained in the description